vips/vip-converter/vip-converter-testnet/vip-converter2.ts

```diff
@@ -0,0 +1,186 @@
+import { ProposalType } from "../../../src/types";
+import { makeProposal } from "../../../src/utils";
+import {
+  BTCBPrimeConverterTokenOuts,
+  BTCB_PRIME_CONVERTER,
+  BaseAssets,
+  CONVERTER_NETWORK,
+  CRITICAL_TIMELOCK,
+  ETHPrimeConverterTokenOuts,
+  ETH_PRIME_CONVERTER,
+  FAST_TRACK_TIMELOCK,
+  NORMAL_TIMELOCK,
+  RISK_FUND_CONVERTER,
+  RiskFundConverterTokenOuts,
+  USDCPrimeConverterTokenOuts,
+  USDC_PRIME_CONVERTER,
+  USDTPrimeConverterTokenOuts,
+  USDT_PRIME_CONVERTER,
+  XVSVaultConverterTokenOuts,
+  XVS_VAULT_CONVERTER,
+} from "./Addresses";
+import {
+  converterCommands,
+  grant,
+  incentiveAndAccessibilityForBTCBPrimeConverter,
+  incentiveAndAccessibilityForETHPrimeConverter,
+  incentiveAndAccessibilityForRiskFundConverter,
+  incentiveAndAccessibilityForUSDCPrimeConverter,
+  incentiveAndAccessibilityForUSDTPrimeConverter,
+  incentiveAndAccessibilityForXVSVaultConverter,
+} from "./commands";
+
+const DEFAULT_PROXY_ADMIN = "0x7877fFd62649b6A1557B55D4c20fcBaB17344C91";
+const RISK_FUND_PROXY = "0x487CeF72dacABD7E12e633bb3B63815a386f7012";
+const RISK_FUND_V2_IMPLEMENTATION = "0x217a907B0c6a7Dc67a21F769a915722B98136F82";
+
+const PROTOCOL_SHARE_RESERVE_PROXY = "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3";
+const PROTOCOL_SHARE_RESERVE_NEW_IMPLEMENTATION = "0x194777360f9DFAA147F462349E9bC9002F72b0EE";
```

Comment:
This implementation does not have the same code as the repo. I would redeploy a new PSR contract from the branch `feat/passive-swap` in the psr repo

Reply:
https://github.com/VenusProtocol/vips/pull/49#discussion_r1446330995
Also, there will be some more changes in implementation due to this:
https://github.com/VenusProtocol/protocol-reserve/pull/9#discussion_r1445506997

Reply:
I would redeploy a new PSR implementation after completing the review of https://github.com/VenusProtocol/protocol-reserve/pull/9

Reply:
Done https://github.com/VenusProtocol/vips/pull/49/commits/ba2b87bec6c7dce5c81e866e45201c043aca6590